src/libraries/System.Private.CoreLib/src/System/Threading/ThreadPool.Portable.cs

```diff
@@ -27,7 +27,7 @@ public static partial class ThreadPool
 #if CORERT
         internal const bool EnableWorkerTracking = false;
```

Comment:
We need to update the field name here as well.

Reply:
Ah yes thanks, will fix